java/core/src/main/java/com/microsoft/bond/Throw.java

```diff
@@ -0,0 +1,180 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+package com.microsoft.bond;
+
+import java.util.Locale;
+
+/**
+ * Encapsulates Bond exception throwing and error messages.
+ * Some message details (e.g. list indices, map keys) may be added to
+ * exception message by certain specialized implementations of Bond type.
+ */
+final class Throw {
+
+    // invariant locale
+    private static final Locale LOCALE = Locale.ROOT;
+
+    // general entry point to raise a data exception on serialization or deserialization
+    static void raiseInvalidDataError(
```

Comment:
Not sure why it's not "Javaism". Is there a "Javaism" authority?

---
In reply to: [123379014](https://github.com/Microsoft/bond/pull/502#discussion_r123379014) [](ancestors = 123379014)